### ISSUE TITLE
Security barrier should report location.

### ISSUE THREAD

**Issue by 00aquir** [yogstation13/yogstation#2196]
It's kinda useless that sec doesn't know where to run just hears that one of the three the barrier spams the chat with messages.

**Comment by GrayRachnid** [comment]
Or you can just rename it depending on it's location

**Comment by 00aquir** [comment]
Apparently you cannot name them more than once. Once you move it elsewhere it's useless.

**Comment by 00aquir** [comment]
Uh. Apparently you cannot change their name if you don't change it entirely.

**Comment by GrayRachnid** [comment]
![dreamseeker_2017-05-15_16-34-51](https://cloud.githubusercontent.com/assets/6613514/26078111/747afa9e-398c-11e7-8e36-b5be2311af57.png)

**Comment by 00aquir** [comment]
Yes this is when that bug happens.

**Comment by Super3222** [comment]
The name you're adding might be too big.

**Comment by 00aquir** [comment]
nah, it's just that it denies any change if you edit it before that part.
